File: 刷题/cpp/2455. 可被三整除的偶数的平均值.cpp
class Solution {
public:
    int averageValue(vector<int>& nums) {
        int sum=0, cnt=0;
        for(auto num:nums){
            if (num % 6 == 0) {
                sum+=num;
                cnt++;
            }
        }
        if (cnt==0) return 0;
        return sum/cnt;
    }
};

File: 刷题/golang/2455. 可被三整除的偶数的平均值.go
func averageValue(nums []int) int {
	sum := 0
	cnt := 0
	for _, num := range nums {
		if num%6 == 0 {
			sum += num
			cnt++
		}
	}
	if cnt == 0 {
		return 0
	}
	return sum / cnt
}